site/reference/cli/alpha/rpkg/copy/README.md

```diff
@@ -19,7 +19,7 @@ revision.
 <!--mdtogo:Long-->
 
 ```
-kpt alpha rpkg copy SOURCE_PACKAGE_REV_NAME TARGET_PACKAGE_NAME [flags]
+kpt alpha rpkg copy SOURCE_PACKAGE_REV_NAME [flags]
```

Comment:
I probably shouldn't update the docs until release. So I'll split this out into another PR.